feedback.md

```diff
@@ -0,0 +1,13 @@
+avoid repeating code and stay DRY(line 52, 53).
+remove redundant code.(line 51 to 53).
+checkJson should include text and not only title
+Ensure that error message are return when there is an error.
+Include a try catch error handling.
+test for the opposite JSON(line)
+In inverted-index-text the falsy is generic so include that it sends an error message(line)
+refactor my multi-word search(line 90, 95)
+Return the word  but the the arry should be empty(line)
+
+
+inverted-index
+there should be a comment above the constructor (line 10)
```

Comment:
I believe this is for your personal consumption and should not be included in your repository.